pmemkv/README.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright (c) 2015 - 2021 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+# PmemKV Driver for YCSB
+This driver is a binding for the YCSB facilities to operate against a [PmemKV](https://github.com/pmem/pmemkv).
+It uses the PmemKV Java bindings.
+
+## Quick Start
+
+### 0. Install PmemKV Java Binding
+**Optionally** you can compile and install custom version of PmemKV Java binding.
+The min. supported version is `1.0.1`, but for better performance at least `1.1.0` is preferred.
```

Comment:
The min. supported is 1.1.0 now, right?

Reply:
no, it actually works just fine with 1.0.1 - but.. we may update this, as you say, to emphasize the usage of the latest release...

Reply:
I've updated that with 1.1.0, to "force" users to use the newest version